lib/commands.ts

```diff
@@ -1,38 +1,48 @@
-import { VisualRegressionTracker } from '@visual-regression-tracker/sdk-js'
+/* global Cypress */
+import { VisualRegressionTracker } from "@visual-regression-tracker/sdk-js";
 
 let vrt: VisualRegressionTracker;
-const { vrt_config } = Cypress.env()
+const { vrt_config } = Cypress.env();
```

Comment:
Codacy found an issue: [Identifier 'vrt_config' is not in camel case.](https://app.codacy.com/gh/Visual-Regression-Tracker/agent-cypress/pullRequest?prid=5918194)